tests/js/server/aql/aql-optimizer-rule-collect-in-cluster.js

```diff
@@ -381,7 +426,134 @@ function optimizerCollectInClusterSingleShardSuite () {
   };
 }
 
+function optimizerCollectInClusterSuiteSmartGraph() {
+  const vn = "UnitTestsVertex";
+  const en = "UnitTestsEdge";
+  const gn = "UnitTestsSmartGraph";
+  const smrt = require("@arangodb/smart-graph");
+
+  return {
+    setUpAll: function () {
+      try {
+        smrt._drop(gn, true);
+      } catch (err) {}
+
+      smrt._create(gn, [smrt._relation(en, vn, vn)], [], { smartGraphAttribute: "group", numberOfShards: 3, isSmart: true });
+
+      let docs = [];
+      for (let i = 0; i < 1000; ++i) {
+        docs.push({group: "test" + (i % 10), value: i});
+      }
+      let keys = db[vn].insert(docs);
+      assertEqual(1000, db[vn].count());
+      
+      docs = [];
+      for (let i = 0; i < 1000; ++i) {
+        docs.push({_from: keys[i]._id, _to: keys[keys.length - i - 1]._id, value: i, group: "test" + (i % 10)});
+      }
+      db[en].insert(docs);
+
+      assertEqual(1000, db[en].count());
+    },
+
+    tearDownAll: function () {
+      smrt._drop(gn, true);
+    },
+
+    testCountSmart: function () {
+      let query = "FOR doc IN " + en + " COLLECT WITH COUNT INTO length RETURN length";
+
+      let results = AQL_EXECUTE(query);
+      assertEqual(1, results.json.length);
+      assertEqual(1000, results.json[0]);
+
+      let plan = AQL_EXPLAIN(query).plan;
+      let nodeTypes = plan.nodes.map(function (node) {
+        return node.type === 'IndexNode' ? 'EnumerateCollectionNode' : node.type;
+      });
+
+      assertNotEqual(-1, plan.rules.indexOf("collect-in-cluster"));
+      assertEqual(["SingletonNode", "EnumerateCollectionNode", "CollectNode", "RemoteNode", "GatherNode", "CollectNode", "ReturnNode"], nodeTypes);
+    },
+
+    testCountMultiSmart: function () {
+      let query = "FOR doc1 IN " + en + " FILTER doc1.value < 10 FOR doc2 IN " + en + " COLLECT WITH COUNT INTO length RETURN length";
+      let results = AQL_EXECUTE(query);
+      assertEqual(1, results.json.length);
+      assertEqual(10000, results.json[0]);
+
+      let plan = AQL_EXPLAIN(query).plan;
+      let nodeTypes = plan.nodes.map(function (node) {
+        return node.type === 'IndexNode' ? 'EnumerateCollectionNode' : node.type;
+      });
+
+      assertNotEqual(-1, plan.rules.indexOf("collect-in-cluster"));
```

Comment:
We could use `includes()` here instead for better readability:
- [Documentation](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/includes?retiredLocale=de)

which then would lead to: 
```
assertFalse(plan.rules.includes("collect-in-cluster"));
```

Or to (positive case) :
```
assertTrue(plan.rules.includes("collect-in-cluster"));
```

Applicable to all occurrences where "indexOf" is being user here.